inst/include/Rcpp/Reference.h

```diff
@@ -59,15 +59,15 @@ namespace Rcpp{
         Reference_Impl( const std::string& klass ) {
             SEXP newSym = Rf_install("new");
             Shield<SEXP> call( Rf_lang2( newSym, Rf_mkString( klass.c_str() ) ) );
-            Storage::set__( Rcpp_eval( call ) ) ;
+            Storage::set__( Rcpp_eval( call , Rcpp::internal::get_Rcpp_namespace()) );
```

Comment:
(I was sort-of-kinda-didn't-quite-say-it complaining about other `methods` cases too; for example nobody understands the nonsense of `Rscript` not calling `methods` by default leading to a bunch of scripts doing funny stuff ...   As I said, I'd just load the bugger -- which is just [what littler does](https://github.com/eddelbuettel/littler/blob/master/autoloads.R#L1).)